src/script/signingprovider.h

```diff
@@ -161,6 +161,7 @@ class SigningProvider
     virtual bool GetKeyOrigin(const CKeyID& keyid, KeyOriginInfo& info) const { return false; }
     virtual bool GetTaprootSpendData(const XOnlyPubKey& output_key, TaprootSpendData& spenddata) const { return false; }
     virtual bool GetTaprootBuilder(const XOnlyPubKey& output_key, TaprootBuilder& builder) const { return false; }
+    virtual std::vector<CPubKey> GetAggregateParticipantPubkeys(const CPubKey& pubkey) const { return {}; }
```

Comment:
Without proper context, it may not be clear that this aggregation specifically refers to a MuSig2 aggregation.
Perhaps a name change could make the purpose and use of the function clearer.
```suggestion
    virtual std::vector<CPubKey> GetMuSig2AggregateParticipantPubkeys(const CPubKey& pubkey) const { return {}; }
```

Reply:
Changed to `GetMuSig2ParticipantPubkeys`